dts/bindings/gpio/ti,launchpad-header.yaml

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2019 Brett Witherspoon
+# SPDX-License-Identifier: Apache-2.0
+
+title: TI LaunchPad GPIO header
+
+description: |
+    This is a representation of GPIO pin nodes exposed as headers on TI LaunchPads
+
```

Comment:
Can you provide more documentation  (cf recent https://github.com/zephyrproject-rtos/zephyr/pull/20555) ?
(a link to TI website would work).
Ideally one day bindings description could be exported to doc.zephyr....

Reply:
I added some documentation modeled after #20555.